core/tests/protractor_utils/CreatorDashboardPage.js

```diff
@@ -73,15 +74,15 @@ var CreatorDashboardPage = function() {
 
   this.navigateToExplorationEditor = async function() {
     await waitFor.elementToBeClickable(explorationDashboardCard);
-    await explorationDashboardCard.click();
+    await action.click('Exploration Dashboard Card', explorationDashboardCard);
```

Comment:
Hi @xarcode, we don't need to do a `waitFor.elementToBeClickable()` for the same element because `action.click` does it internally. Can you remove the `waitFor.elementToBeClickable()` statement here and in other places you have made changes? Thanks!

Reply:
Thanks for pointing it out! I will fix it as soon as possible!